openstf/feature_engineering/apply_features.py

```diff
@@ -149,185 +55,50 @@ def apply_features(data, minute_list=None, h_ahead=24):
                             np.random.uniform(0.7,1.7, 200)))
 
     """
-    if minute_list is None:
-        minute_list = []
 
-    lag_functions = generate_lag_functions(data, minute_list, h_ahead)
+    # Get lag feature functions
+    lag_feature_functions = generate_lag_feature_functions(feature_set_list, horizon)
 
-    # TODO it seems not all feature use the same convention
-    # better to choose one and stick to it
-    timedriven_functions = {
+    # Get timedrivenfeature functions
+    timedriven_feature_functions = {
         "IsWeekendDay": lambda x: (x.index.weekday // 5) == 1,
         "IsWeekDay": lambda x: x.index.weekday < 5,
-        "IsMonday": lambda x: x.index.weekday == 0,
```

Comment:
@JanMaartenvanDoorn Why where these removed here?

Reply:
These features are not included in the feature lists